addons/web/static/src/views/fields/relational_utils.js

```diff
@@ -338,6 +338,14 @@ export class Many2XAutocomplete extends Component {
             abort: originalPromise.abort ? originalPromise.abort.bind(originalPromise) : () => {},
         };
     }
+
+    get searchSpecifications() {
```

Comment:
For my information, what's this method for? (and what is specification?) :slightly_smiling_face:

Reply:
I don't have a complete understanding of the what and why, but it's used to determine what fields are going to be fetched from the database. I created a getter to override the "specification" in hr_holiday and fetch one more field.

I don't know why this is called "specification" and why it's an object and not a list of strings is beyond me.